packages/core/src/utilities/loadImageToCanvas.ts

```diff
@@ -0,0 +1,86 @@
+import { IImage } from '../types';
+
+import { loadAndCacheImage } from '../imageLoader';
+import * as metaData from '../metaData';
+import { RequestType } from '../enums';
+import imageLoadPoolManager from '../requestPool/imageLoadPoolManager';
+import renderToCanvas from './renderToCanvas';
+
+/**
+ * Loads and renders an imageId to a Canvas. It will use the CPU rendering pipeline
+ * for image.
+ *
+ * @example
+ * ```
+ * const canvas = document.getElementById('myCanvas')
+ * const imageId = 'myImageId'
+ *
+ * loadImageToCanvas(canvas, imageId)
+ * ```
+ * @param imageId - The imageId to render
+ * @param canvas - Canvas element to render to
+ * @param requestType - The type of request (default to interaction), can be 'interaction' or 'prefetch' or 'thumbnail'
+ * the order of loading for the pool manager is interaction, thumbnail, prefetch
+ * @param priority - The priority of the request within the request type (lower is higher priority)
+ * @returns - A promise that resolves when the image has been rendered with the imageId
+ */
+export default function loadImageToCanvas(
+  canvas: HTMLCanvasElement,
+  imageId: string,
+  requestType = RequestType.Interaction,
+  priority = -5
```

Comment:
Shall we just hardcode this as something super high priority? It feels like no one, not even power users, would ever specify anything other than _now_ for this? Setting a priority so it loads in some queue you have no knowledge of when making this call feels realllly weird.

Reply:
I think it has some value to keep the parameters and give default options.